package.json

```diff
@@ -82,8 +81,11 @@
     "check:deps": "lerna exec --no-bail --stream depcheck",
     "check:peer-deps": "check-peer-dependencies",
     "check:typescript-project-references": "workspaces-to-typescript-project-references --includesRoot --check",
-    "vscode:package": "lerna exec --scope @salesforce/salesforcedx-* -- npm prune --production && lerna run vscode:package --concurrency 1 && npm run reformat",
-    "vsix:install": "find ./packages -name '*.vsix' -exec code-insiders --install-extension {} \\;",
+    "vscode:bundle": "npm run vscode:bundle:debugger && npm run vscode:bundle:extension",
+    "vscode:bundle:debugger": "lerna run bundle:debugger",
+    "vscode:bundle:extension": "lerna run bundle:extension",
+    "vscode:package": "npm run vscode:bundle:debugger && npm run vscode:bundle:extension && lerna run vscode:package --concurrency 1 && npm run reformat",
+    "vsix:install": "find ./packages -name '*.vsix' -exec code --install-extension {} \\;",
```

Comment:
Hum..I didn't intent to commit this change to update this script to install on insiders.  Does anyone use this?